training/distributed_training/pytorch/model_parallel/gpt-j/01_train_gptj_smp_notebook.ipynb

```diff
@@ -2,6 +2,7 @@
  "cells": [
```

Comment:
<p>Is this using a Feature Store variable from another notebook? If so, explicitly mention which notebook and what/why it's doing.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3360/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
removed dependency